app/views/lessons/show.jbuilder

```diff
@@ -3,4 +3,6 @@ json.post do
   json.title @lesson.title
   json.description @lesson.description
   json.video_link @lesson.video_link
+  json.author_id @lesson.author_id
```

Comment:
You need to remove 'post' from here. Now you return something like this:

{
  "post": {
    "id": 6,
    "title": "string",
    "description": "string",
    "video_link": "string",
    "author_id": 1,
    "category_id": 1
  }
}

See app/views/categories/show.jbuilder as example